ui/app/components/clients/activity.ts

```diff
@@ -108,53 +108,4 @@ export default class ClientsActivityComponent extends Component<Args> {
     const { versionHistory, activity } = this.args;
     return filterVersionHistory(versionHistory, activity.startTime, activity.endTime);
   }
-
-  // (object) single month new client data with total counts and array of
-  // either namespaces or mounts
-  get newClientCounts() {
```

Comment:
These were only used for overview component, so I moved them and then they got overwritten with the other changes